clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/controllers/GoogleLoadBalancerController.groovy

```diff
@@ -104,17 +104,21 @@ class GoogleLoadBalancerController {
       }
     }
 
+    def isHttp = view.loadBalancerType == GoogleLoadBalancerType.HTTP
     [new GoogleLoadBalancerDetails(loadBalancerName: view.name,
+                                   loadBalancerType: view.loadBalancerType,
                                    createdTime: view.createdTime,
                                    dnsname: view.ipAddress,
                                    ipAddress: view.ipAddress,
                                    healthCheck: view.healthCheck ?: null,
                                    backendServiceHealthChecks: backendServiceHealthChecks ?: null,
                                    listenerDescriptions: [[
-                                       listener: new ListenerDescription(instancePort: view.portRange,
-                                                                         loadBalancerPort: view.portRange,
-                                                                         instanceProtocol: view.ipProtocol,
-                                                                         protocol: view.ipProtocol)
+                                       listener: new ListenerDescription(
+                                         instancePort: isHttp ? 'http' : Utils.getPortFromPortRange(view.portRange), // Https forwards traffic to a 'named' port.
```

Comment:
Did you test that this has no negative implications on the Edit Load Balancer capability in the ui?